### PR TITLE
docs(contribution guide): add requirements about commit and PR naming conventions

### PR DESCRIPTION
**Pull request by grzesuav**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Pull Request template
+Please, go through these steps before you submit a PR.
+
+1. This repository follows semantic versioning convention, therefore each PR title/commit message must follow convention: `<type>(<scope>): <subject>`.
```
Comment (AmitKumarDas) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: thanks @grzesuav 

Will it be possible to add a tip or two about `subject` as well.
For example, following are valid:
- `#issueno short title` or
- `short title`

Comment (AmitKumarDas) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: refer to this [commit](https://github.com/AmitKumarDas/metac/pull/132/commits/7678e907a4756e70584a1beee1792f2923f7bd60)

We may ignore if above is not valid as per semantic conventions

Comment (grzesuav) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: will add this later today, good idea :+1:

Comment (grzesuav) [reply .github/PULL_REQUEST_TEMPLATE.md]: added

**Pull request merged**